src/main/java/core/basesyntax/dao/ReadFromFile.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.dao;
+
+import com.opencsv.CSVParser;
+import com.opencsv.CSVParserBuilder;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import com.opencsv.exceptions.CsvException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+public class ReadFromFile {
```

Comment:
```suggestion
public class FileReader {
```

Reply:
I have FileReder from java.io so I renamed it to ShopFileReader or I have one more idea to change it to FruitFileReader.